### ISSUE TITLE
Running Strive without GG-Struggle seems to mess up the game and wont let it even connect to play online.

### ISSUE THREAD

**Issue by GPAngelOracle** [tsaibermelon/gg-struggle#22]
Not sure if this counts as an issue, but no one has brought it up so i felt i might aswell, so when i run the game without opening GG-Struggle it gives me this error message. Uninstalling does seem to let the game run normally. 
![image](https://user-images.githubusercontent.com/89692208/131219167-90cecf15-bd9f-46fa-9281-3ca8bc6ba170.png)

**Comment by cybermelons** [comment]
Yep. That’s expected behavior since gg struggle sits between the game and the game servers. If you don’t run it, you’re cutting off the connection in the middle resulting in that error
